lectia-3.js

```diff
@@ -1,2 +1,65 @@
-// https://miorita.scoalaweb.ro/lessons/3-while
-// Scrie un program care să o ajute pe Miorița să ajungă la pășunea din centrul spiralei
+// Varianta lunga
+/* 
+reset();
+for(let i = 0; i < 6; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 6; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 6; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 5; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 5; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 4; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 4; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 3; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 3; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 2; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 2; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 1; i++) {
+    move();
+}
+turnRight();
+for(let i = 0; i < 1; i++) {
+    move();
+} */
+// Varianta scurta
+reset();
+let list = [6, 6, 6, 5, 5, 4, 4, 3, 3, 2, 2, 1];
```

Comment:
Există și alte metode de a rezolva problema asta, nu doar cu array.

Reply:
Dupa multe incercari cu ce aveam în suportul de curs și în cap ma apropiam de soluția asta, însă era mai lungă.
Apoi am găsit canalul ScoalaWeb și am vazut tutorialul și am reușit. Practic tot cu ajutor am rezolvat labirintul, dar am și inteles :D

Reply:
Oh No! My secret is out 😱

Reply:
Ups, ce să fac dacă Google și Youtube sunt soluția pentru orice problema am.🤦‍♀️ 🤣  La celalalt nu m-am uitat, aștept cursul de azi. 😄

Reply:
> Google și Youtube sunt soluția pentru orice problema am

Înseamnă că te comporți deja deja ca un programator.